packages/db/package.json

```diff
@@ -9,6 +9,10 @@
     "db-dump:prd": "docker exec oak-ai-beta pg_dump $(doppler secrets get -c prd --plain DATABASE_ADMIN_URL | sed -e 's/?sslmode=required//g') --no-owner --exclude-table-data public.generations --exclude-table-data public.snippets --exclude-table-data public.downloads",
     "db-dump:stg": "docker exec oak-ai-beta pg_dump $(doppler secrets get -c stg --plain DATABASE_ADMIN_URL | sed -e 's/?sslmode=required//g') --no-owner --exclude-table-data public.generations --exclude-table-data public.snippets --exclude-table-data public.downloads",
     "db-dump:stg:homebrew": "pg_dump $(doppler secrets get -c stg --plain DATABASE_ADMIN_URL | sed -e 's/?sslmode=required/?sslcert=./certs/db_stg.pem/g') --no-owner --exclude-table-data public.generations --exclude-table-data public.snippets --exclude-table-data public.downloads",
+    "db-seed-local-from:stg": "doppler run --config stg -- bash -c 'source scripts/import-export/db_seed.sh stg dev'",
+    "db-seed-local-from:prd": "doppler run --config prd -- bash -c 'source scripts/import-export/db_seed.sh prd dev'",
+    "db-seed-stg-from:local": "doppler run --config stg -- bash -c 'source scripts/import-export/db_seed.sh dev stg'",
+    "db-seed-prd-from:local": "doppler run --config prd -- bash -c 'source scripts/import-export/db_seed.sh dev prd'",
```

Comment:
Little bit weary about having this script just casually there!

Reply:
Same, the original idea behind this PR was to do with pushing the locally ingested data up to live environments. If your ingest PRs actually solve this problem then we could remove those and have this functionality focusing on seeding dev environments from live?